src/server/zset_family.cc

```diff
@@ -3004,6 +3004,118 @@ void ZSetFamily::GeoSearch(CmdArgList args, ConnectionContext* cntx) {
   GeoSearchStoreGeneric(cntx, shape, key, member, geo_ops);
 }
 
+void ZSetFamily::GeoRadius(CmdArgList args, ConnectionContext* cntx) {
```

Comment:
That's a lot of code! 🙂 

Quoting Redis docs:
> It can be replaced by [GEOSEARCH](https://redis.io/docs/latest/commands/geosearch/) and [GEOSEARCHSTORE](https://redis.io/docs/latest/commands/geosearchstore/) with the BYRADIUS and FROMMEMBER arguments when migrating or writing new code.

So they already hint that GeoRadius is just a specification of GeoSearch. Maybe we can unite them somehow? They share count, asc/desc, withcord, withdist, storedist... Not mentioning parsing lon/lat and units

Reply:
> That's a lot of code! 🙂
> 
> Quoting Redis docs:
> 
> > It can be replaced by [GEOSEARCH](https://redis.io/docs/latest/commands/geosearch/) and [GEOSEARCHSTORE](https://redis.io/docs/latest/commands/geosearchstore/) with the BYRADIUS and FROMMEMBER arguments when migrating or writing new code.
> 
> So they already hint that GeoRadius is just a specification of GeoSearch. Maybe we can unite them somehow? They share count, asc/desc, withcord, withdist, storedist... Not mentioning parsing lon/lat and units

Thanks for reviewing. Updated as you suggested.